diff -Naur poco-poco-1.5.2-release.orig/Foundation/src/Error.cpp poco-poco-1.5.2-release/Foundation/src/Error.cpp
--- poco-poco-1.5.2-release.orig/Foundation/src/Error.cpp	2013-09-18 15:12:27.000000000 -0300
+++ poco-poco-1.5.2-release/Foundation/src/Error.cpp	2013-12-18 08:08:49.000000000 -0200
@@ -75,10 +75,10 @@
 		   without -D_GNU_SOURCE is needed, otherwise the GNU version is
 		   preferred.
 		*/
-#ifdef _GNU_SOURCE
+#if defined(_GNU_SOURCE) && !defined(POCO_ANDROID)
 		char errmsg[256] = "";
 		return std::string(strerror_r(errorCode, errmsg, 256));
-#elif (_XOPEN_SOURCE >= 600)
+#elif (_XOPEN_SOURCE >= 600) || defined(POCO_ANDROID)
 		char errmsg[256] = "";
 		strerror_r(errorCode, errmsg, 256);
 		return errmsg;
