diff -Naur poco-poco-1.5.2-release.orig/Foundation/include/Poco/AtomicCounter.h poco-poco-1.5.2-release/Foundation/include/Poco/AtomicCounter.h
--- poco-poco-1.5.2-release.orig/Foundation/include/Poco/AtomicCounter.h	2013-09-18 15:12:27.000000000 -0300
+++ poco-poco-1.5.2-release/Foundation/include/Poco/AtomicCounter.h	2013-12-18 08:04:19.000000000 -0200
@@ -53,6 +53,10 @@
 	#if !defined(POCO_HAVE_GCC_ATOMICS) && !defined(POCO_NO_GCC_ATOMICS)
 		#define POCO_HAVE_GCC_ATOMICS
 	#endif
+#elif ((__clang_major__ == 3) && (__clang_minor__ >= 1) || __clang_major__ > 3)
+	#if !defined(POCO_HAVE_GCC_ATOMICS) && !defined(POCO_NO_GCC_ATOMICS)
+		#define POCO_HAVE_GCC_ATOMICS
+	#endif
 #endif // POCO_OS
 #include "Poco/Mutex.h"
 
