diff -Naur poco-poco-1.5.2-release.orig/Makefile-custom poco-poco-1.5.2-release/Makefile-custom
--- poco-poco-1.5.2-release.orig/Makefile-custom	1969-12-31 21:00:00.000000000 -0300
+++ poco-poco-1.5.2-release/Makefile-custom	2014-01-20 16:06:14.000000000 -0200
@@ -0,0 +1,79 @@
+##############################################################################
+# Customizations for more fine-grained control over Makefile by libopencash  #
+##############################################################################
+
+# custom targets
+
+my_static_debug:
+	$(MAKE) -f Makefile -f ../Makefile-custom -C $(POCO_BASE)/Foundation my_lib_static_debug
+
+my_static_release:
+	$(MAKE) -f Makefile -f ../Makefile-custom -C $(POCO_BASE)/Foundation my_lib_static_release
+
+my_shared_debug:
+	$(MAKE) -f Makefile -f ../Makefile-custom -C $(POCO_BASE)/Foundation my_lib_shared_debug
+
+my_shared_release:
+	$(MAKE) -f Makefile -f ../Makefile-custom -C $(POCO_BASE)/Foundation my_lib_shared_release
+
+# Override the debug libraries to _not_ append the 'd' suffix
+
+MY_LIBPATH_DEBUG           = $(LIBPATH)/debug
+MY_LIB_DEBUG_STATIC        = $(MY_LIBPATH_DEBUG)/$(LIBPREFIX)$(target)$(OSARCH_POSTFIX).a
+MY_LIB_DEBUG_SHARED        = $(MY_LIBPATH_DEBUG)/$(LIBPREFIX)$(target)$(OSARCH_POSTFIX)$(SHL_EXT)
+MY_LIB_DEBUG_SHARED_LINK   = $(MY_LIBPATH_DEBUG)/$(LIBPREFIX)$(target)$(OSARCH_POSTFIX)$(SHAREDLIBLINKEXT)
+
+MY_LIBPATH_RELEASE         = $(LIBPATH)/release
+MY_LIB_RELEASE_STATIC      = $(MY_LIBPATH_RELEASE)/$(LIBPREFIX)$(target)$(OSARCH_POSTFIX).a
+MY_LIB_RELEASE_SHARED      = $(MY_LIBPATH_RELEASE)/$(LIBPREFIX)$(target)$(OSARCH_POSTFIX)$(SHL_EXT)
+MY_LIB_RELEASE_SHARED_LINK = $(MY_LIBPATH_RELEASE)/$(LIBPREFIX)$(target)$(OSARCH_POSTFIX)$(SHAREDLIBLINKEXT)
+
+$(MY_LIBPATH_DEBUG) $(MY_LIBPATH_RELEASE):
+	$(MKDIR) $@
+
+my_lib_static_debug:   libdirs $(MY_LIBPATH_DEBUG) $(MY_LIB_DEBUG_STATIC)
+
+my_lib_static_release: libdirs $(MY_LIBPATH_RELEASE) $(MY_LIB_RELEASE_STATIC)
+
+my_lib_shared_debug:   libdirs $(MY_LIBPATH_DEBUG) $(MY_LIB_DEBUG_SHARED)
+
+my_lib_shared_release: libdirs $(MY_LIBPATH_RELEASE) $(MY_LIB_RELEASE_SHARED)
+
+$(MY_LIB_DEBUG_STATIC): $(foreach o,$(objects),$(OBJPATH_DEBUG_STATIC)/$(o).o)
+	@echo "** Building library (debug)" $@
+	$(LIB) $@ $^
+	$(RANLIB) $@
+	$(postbuild)
+
+$(MY_LIB_RELEASE_STATIC): $(foreach o,$(objects),$(OBJPATH_RELEASE_STATIC)/$(o).o)
+	@echo "** Building library (release)" $@
+	$(LIB) $@ $^
+	$(RANLIB) $@
+	$(postbuild)
+
+$(MY_LIB_DEBUG_SHARED): $(foreach o,$(objects),$(OBJPATH_DEBUG_SHARED)/$(o).o)
+	@echo "** Building library (debug)" $@
+	$(LIB) $@ $^
+	$(RANLIB) $@
+	$(postbuild)
+
+$(MY_LIB_RELEASE_SHARED): $(foreach o,$(objects),$(OBJPATH_RELEASE_SHARED)/$(o).o)
+	@echo "** Building library (release)" $@
+	$(LIB) $@ $^
+	$(RANLIB) $@
+	$(postbuild)
+
+my_install:
+	mkdir -p $(INSTALLDIR)/include/Poco
+	mkdir -p $(INSTALLDIR)/lib
+	mkdir -p $(INSTALLDIR)/bin
+	for comp in $(COMPONENTS) ; do \
+		if [ -d "$(POCO_BASE)/$$comp/include" ] ; then \
+			cp -Rf $(POCO_BASE)/$$comp/include/* $(INSTALLDIR)/include/ ; \
+		fi ; \
+		if [ -d "$(POCO_BUILD)/$$comp/bin" ] ; then \
+			find $(POCO_BUILD)/$$comp/bin -perm -700 -type f -exec cp -f {} $(INSTALLDIR)/bin \; ; \
+		fi ; \
+	done
+	find $(POCO_BUILD)/lib -name "libPoco*" -type f -exec cp -f {} $(INSTALLDIR)/lib \;
+	find $(POCO_BUILD)/lib -name "libPoco*" -type l -exec cp -Rf {} $(INSTALLDIR)/lib \;
